packages/web-components/fast-components/package.json

```diff
@@ -92,6 +92,7 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.0.8",
     "@microsoft/fast-element": "^0.8.1",
+    "@microsoft/fast-foundation": "^0.0.0",
     "@microsoft/fast-web-utilities": "^4.4.5",
     "tabbable": "^4.0.0"
```

Comment:
we should now be able to remove any dependencies that fast-foundation also takes, such as tabbable

Reply:
correct - good call. Although, if we reference something like `fast-web-utilities` or `fast-element` directly will we still want those?

Reply:
as dependencies? we shouldn't need to add them in the package json even if we references them.

Reply:
Per our discussion ,if they are directly referenced then explicitly add them.